### ISSUE TITLE
actor loss being used instead of critic loss to determine burn in

### ISSUE THREAD

**Issue by tomsilver** [k-r-allen/residual-policy-learning#14]
In the write up for this work (https://arxiv.org/abs/1812.06298) we said:

"We therefore propose to train the critic alone for a “burn in” period  while  leaving  the  policy  fixed.  We  can  determine  an appropriate  burn  in  length  automatically  by  monitoring  the critic loss function and waiting for it to dip below a threshold β,  which  becomes  a  hyperparameter  of  our  method."

In the code, though, we actually monitor the actor loss, rather than the critic loss, to determine the burn in period:

- https://github.com/k-r-allen/residual-policy-learning/blob/master/tensorflow/experiment/train_staged.py#L68
- https://github.com/k-r-allen/residual-policy-learning/blob/master/tensorflow/experiment/train_staged.py#L97

Also see this mismatch between actor/critic loss:

- https://github.com/k-r-allen/residual-policy-learning/blob/master/tensorflow/experiment/train_staged.py#L98

We should fix this and see if it changes performance. If actor loss is actually better than critic loss, we should update the arXiv paper.
